README.md

```diff
@@ -17,4 +22,24 @@ If you want to dive right into some code for extending the debug engine, take a
 * [Managed Expression Evaluator](https://github.com/Microsoft/ConcordExtensibilitySamples/wiki/Managed-Expression-Evaluator-Sample)
 * [C++ Custom Visualizer](https://github.com/Microsoft/ConcordExtensibilitySamples/wiki/Cpp-Custom-Visualizer-Sample)
 
+## Documentation Generation
```

Comment:
I feel like we should either move this to the Debugger OneNote or at least to DOCFX_USAGE.md since no customer is going to care about this